integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/rest/test/header/HTTPResponseCodeTestCase.java

```diff
@@ -41,69 +41,74 @@
 import org.wso2.esb.integration.common.utils.ESBTestCaseUtils;
 
 public class HTTPResponseCodeTestCase extends ESBIntegrationTest {
+
     private Log log = LogFactory.getLog(HTTPResponseCodeTestCase.class);
-	private HttpServer server = null;
+    private HttpServer server = null;
+    private static int responseCode;
```

Comment:
Why this is declared as static?

Reply:
Because it should be referenced from a static content

Reply:
Usages of responseCode does not show a static method. Shall we correct?

Reply:
Done